### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by Soykaa**
Сriticism is welcome :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<img src="logo.png" align="right" />
+
+# UCheck
+<!-- Banges -->
+[![releases](https://img.shields.io/github/v/release/DLochmelis33/UCheck.svg)](https://github.com/DLochmelis33/UCheck/blob/master/CHANGELOG.md)
+[![MIT License Badge][license-badge]][license]
```
Comment (GlebSolovev) [inline README.md]: Displayed incorrectly :(
image

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+[![Java CI with Gradle](https://img.shields.io/github/workflow/status/DLochmelis33/UCheck/Java%20CI%20with%20Gradle)](https://github.com/DLochmelis33/UCheck/actions/workflows/on_push.yml)
+[![contributions](https://img.shields.io/github/contributors/DLochmelis33/UCheck)](https://github.com/DLochmelis33/UCheck/graphs/contributors)
+![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)
+## Introduction
+UCheck is created to help people buy a variety of goods at the lowest prices. 
+
+### What can you do with UCheck?
+* Scan your check after shopping
+* Agregate all purchase information (prices, items, shop, etc) in UCheck database
+* Use the collected infornation to find necessary goods at the lowest prices
```
Comment (GlebSolovev) [inline README.md]: Typo: `infornation`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+2. Execute Gradle
+
+## Authors
+- Gleb Solovev ([@GlebSolovev](https://github.com/GlebSolovev))
+- Denis Lochmelis ([@DLochmelis33](https://github.com/DLochmelis33))
+- Ekaterina Itsenko ([@Soykaa](https://github.com/Soykaa))
+
+## Acknowledgements
+We are very grateful to our wonderful teachers for their support and assistance during the whole project:
```
Comment (GlebSolovev) [inline README.md]: Very very a lot super mega **grateful**!

File: README.md (hunk 1 above)
Comment (GlebSolovev) [reply README.md]: And also, we have another license &mdash; GPL instead of MIT. You can find out the details from @DLochmelis33

File: README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- Vladislav Tankov
+- Timofey Bryksin
+
+## Contribute
+Contributions are welcome! Before creating a pull request, make sure that it adheres to the following guidelines:
+
+- Alphabetize your entry
```
Comment (GlebSolovev) [inline README.md]: What does this mean?

**Review by GlebSolovev — COMMENTED** [review]
Good work! But something needs to be fixed.

File: README.md (hunk 1 above)
Comment (Soykaa) [reply README.md]: Hope that now it works (???)

File: README.md (hunk 3 above)
Comment (Soykaa) [reply README.md]: if it's not a joke.........

File: README.md (hunk 4 above)
Comment (Soykaa) [reply README.md]: hm, nevermind, I think it's useless, my mistake

File: README.md (hunk 2 above)
Comment (Soykaa) [reply README.md]: fixed

File: README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+<img src="logo.png" align="right" />
+
+# UCheck
+<!-- Banges -->
```
Comment (DLochmelis33) [inline README.md]: Badges?

**Review by DLochmelis33 — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (Soykaa) [reply README.md]: oops, fixed, sorry

**Pull request merged**